omisego-sdk/src/main/java/co/omisego/omisego/websocket/channel/dispatcher/CustomEventDispatcher.kt

```diff
@@ -0,0 +1,140 @@
+package co.omisego.omisego.websocket.channel.dispatcher
+
+/*
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 12/5/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+
+import co.omisego.omisego.model.APIError
+import co.omisego.omisego.model.socket.SocketReceive
+import co.omisego.omisego.model.transaction.consumption.TransactionConsumption
+import co.omisego.omisego.websocket.SocketChannelCallback
+import co.omisego.omisego.websocket.SocketCustomEventCallback
+import co.omisego.omisego.websocket.enum.SocketCustomEvent
+import co.omisego.omisego.websocket.enum.SocketCustomEvent.OTHER
+import co.omisego.omisego.websocket.enum.SocketCustomEvent.TRANSACTION_CONSUMPTION_FINALIZED
+import co.omisego.omisego.websocket.enum.SocketCustomEvent.TRANSACTION_CONSUMPTION_REQUEST
+
+/**
+ * A callback for dispatch the [SocketCustomEventCallback] events.
+ */
+class CustomEventDispatcher : SocketDispatcherContract.CustomEventDispatcher {
+    /**
+     * For dispatching the [SocketCustomEventCallback] event.
+     */
+    override var socketCustomEventCallback: SocketCustomEventCallback? = null
+
+    /**
+     * The web socket replied object from eWallet API.
+     */
+    override var socketReceive: SocketReceive? = null
+
+    /**
+     * For dispatching the [SocketChannelCallback] event.
+     */
+    override var socketChannelCallback: SocketChannelCallback? = null
+
+    /**
+     * Handles the [SocketCustomEvent] and dispatch the [SocketCustomEventCallback] to the client.
+     *
+     * @param customEvent To indicate the actual type of generic [SocketCustomEvent]
+     */
+    override fun handleEvent(customEvent: SocketCustomEvent) {
+        val response = socketReceive ?: return
+        val listenEvent = socketCustomEventCallback ?: return
+
+        when (listenEvent) {
+            is SocketCustomEventCallback.TransactionRequestCallback ->
```

Comment:
Is it possible to import the class and thus write `is TransactionRequestCallback` ? 
In this scope it wouldn't be an issue, no ambiguity and would make the when block much more readable